internal/settings/settings.go

```diff
@@ -21,10 +21,9 @@ type Options struct {
 	// ExperimentalFeatures encapsulates experimental features users can opt into.
 	ExperimentalFeatures ExperimentalFeatures `mapstructure:"experimentalFeatures"`
 
-	// TODO: Need to check for conflict with CLI flags
-	TerraformExecPath string `mapstructure:"terraformExecPath"`
-	// TerraformExecTimeout time.Duration
-	// TerraformLogFilePath string
+	TerraformExecPath        string `mapstructure:"terraformExecPath"`
+	TerraformExecTimeout     string `mapstructure:"terraformExecTimeout"`
+	TerraformExecLogFilePath string `mapstructure:"terraformExecLogFilePath"`
```

Comment:
`TerraformExecLogFilePath` changed var name here to keep it consistent, feel free to update this though.

Reply:
Oh, knowing [`terraform-exec`](https://github.com/hashicorp/terraform-exec) is a thing now, it clears a lot of things up. I was under the impression that they all are being passed on to the terraform binary one way or the other. Thanks for clearing this up.